CMakeLists.txt

```diff
@@ -5,7 +5,6 @@ if(NOT DEFINED CMAKE_MACOSX_RPATH)
     set(CMAKE_MACOSX_RPATH 0)
 endif()
 
-
 project(ChronoLog LANGUAGES CXX)
```

Comment:
For me, I have to add C to the language list to build those MPI-based test cases.